packages/wrangler/src/d1/timeTravel/restore.ts

```diff
@@ -101,7 +101,7 @@ const handleRestore = async (
 	searchParams: URLSearchParams
 ) => {
 	return await fetchResult<RestoreBookmarkResponse>(
-		`/accounts/${accountId}/d1/database/${databaseId}/time-travel/restore?${searchParams.toString()}`,
+		`/accounts/${accountId}/d1/database/${databaseId}/time_travel/restore?${searchParams.toString()}`,
```

Comment:
Not directly related to this PR - but it's a shame that lots of these D1 commands don't have any tests in Wrangler.